### PR TITLE
Added info about withdrawn_from, withdrawal labels

### PR DESCRIPTION
**Pull request by vterentev**
## Checklist

- [ ] My article available in Navigation sidebar and in root article of selected category ([how to do it in README](https://github.com/santiment/academy#add-an-article-into-navigation-sidebar))
- [ ]	My article have [metadata](https://github.com/santiment/academy#metadata) (title, description, date when updated and author)
- [ ]	All added/updated links in article are exist, images shows correctly

### CODE REVIEW

File: src/docs/labels/withdrawal/index.md [hunk 1]
```diff
@@ -6,9 +6,20 @@ description: label
 ---
 
 ## Description
+A withdrawal address is and address that belongs to a user and used by a user in order to withdraw funds from a third-party, e.g CEX.
```
Comment (IvanIvanoff) [inline src/docs/labels/withdrawal/index.md:9]:
```suggestion
A withdrawal address is an address that belongs to a user and is used by a user in order to withdraw funds from a third-party (e.g CEX, DEX).
```

File: src/docs/labels/withdrawal/index.md [hunk 2]
```diff
@@ -6,9 +6,20 @@ description: label
...
+The address is considered as a withdrawal address if it received funds from CEX (withdraw).
```
Comment (IvanIvanoff) [inline src/docs/labels/withdrawal/index.md:10]:
```suggestion
The address is considered as a withdrawal address if it has received funds from a CEX (withdraw).
```
Is this only CEX? What about DEX?

File: src/docs/labels/withdrawal/index.md [hunk 3]
```diff
@@ -6,9 +6,20 @@ description: label
...
+
+Most common case is a centralized exchange withdrawal. See the details on the picture below.
```
Comment (IvanIvanoff) [inline src/docs/labels/withdrawal/index.md:12]:
```suggestion
The most common case is a centralized exchange (CEX) withdrawal. See the details in the image below.
```

File: src/docs/labels/withdrawn-from/index.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+title: Withdrawn From
+author: Vladimir
+date: 2024-04-08
+description: label
+---
+
+## Description
+A "withdrawn_from" label is used to identify the entity involved in a withdrawal action.
```
Comment (IvanIvanoff) [inline src/docs/labels/withdrawn-from/index.md:9]:
```suggestion
A`withdrawn_from` label is used to identify the entity involved in a withdrawal action.
```

File: src/docs/labels/withdrawn-from/index.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+Most common case is a centralized exchange withdrawal. See the details on the picture below.
```
Comment (IvanIvanoff) [inline src/docs/labels/withdrawn-from/index.md:11]:
```suggestion
The most common case is a centralized exchange (CEX) withdrawal. See the details in the image below.
```

**Review by alexander-gr — APPROVED** [review]

**Pull request merged**